src/components/UI/MeatBalls.jsx

```diff
@@ -0,0 +1,52 @@
+import { MoreHoriz, MoreVert } from '@mui/icons-material'
+import { IconButton, Menu, MenuItem, styled } from '@mui/material'
+import React, { useState } from 'react'
+
+export const MeatBalls = ({ variant = 'horiz', handleChange, options }) => {
+   const [anchorEl, setAnchorEl] = useState(null)
+   const open = Boolean(anchorEl)
+   const handleClick = (event) => {
+      setAnchorEl(event.currentTarget)
+   }
+   const handleClose = () => {
+      setAnchorEl(null)
+   }
+   return (
+      <>
+         <IconButton onClick={handleClick}>
+            {variant === 'horiz' ? <MoreHoriz /> : <MoreVert />}
+         </IconButton>
+         <StyledMenu anchorEl={anchorEl} open={open} onClick={handleClose}>
+            {options.map(({ title, icon }) => (
+               <StyledMenuItem
+                  key={title}
+                  onClick={(e) => {
+                     handleClose()
+                     handleChange(e)
+                  }}
+                  value={title}
+               >
+                  <StyledImage src={icon} alt={title} />
```

Comment:
![image](https://github.com/peaksoft-school/gift-list-js10/assets/130536609/0de16bef-ccf9-4a89-866c-50ef8e28e6cf)
there should be an icon here, not a picture